### ISSUE TITLE
ERROR: matplotlib.tests.test_patheffects.test_PathEffect_get_proxy

### ISSUE THREAD

**Issue by jlec** [matplotlib/matplotlib#5291]
```python
======================================================================
ERROR: matplotlib.tests.test_patheffects.test_PathEffect_get_proxy
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/matplotlib-1.4.3/work/matplotlib-1.4.3-python3_5/build/test/lib/matplotlib-1.4.3-py3.5-linux-x86_64.egg/matplotlib/testing/decorators.py", line 110, in wrapped_function
    func(*args, **kwargs)
  File "/var/tmp/portage/dev-python/matplotlib-1.4.3/work/matplotlib-1.4.3-python3_5/build/test/lib/matplotlib-1.4.3-py3.5-linux-x86_64.egg/matplotlib/tests/test_patheffects.py", line 96, in test_PathEffect_get_proxy
    dep.assert_called()
  File "/usr/lib64/python3.5/unittest/mock.py", line 583, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called

======================================================================
ERROR: matplotlib.tests.test_patheffects.test_SimplePatchShadow_offset_xy
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/var/tmp/portage/dev-python/matplotlib-1.4.3/work/matplotlib-1.4.3-python3_5/build/test/lib/matplotlib-1.4.3-py3.5-linux-x86_64.egg/matplotlib/tests/test_patheffects.py", line 123, in test_SimplePatchShadow_offset_xy
    dep.assert_called()
  File "/usr/lib64/python3.5/unittest/mock.py", line 583, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called

----------------------------------------------------------------------
```
python-3.5
matplotlib-1.4.3

**Comment by QuLogic** [comment]
I believe this is fixed on master and the 1.5.x release candidates. Can you try with one of those?

**Comment by jlec** [comment]
Exactly, fixed in f98c4846dc3c15b3d24aafb973764cb9b860d935
